server/lib/packets/packet.go

```diff
@@ -92,16 +103,39 @@ func (pkt *RecvPacket) Process(t Transport) {
 	}
 }
 
+func Actor2ProtoError(aerr *yue.ActorError) *proto.Err {
+	if aerr.Is(yue.ActorUnavailable) {
```

Comment:
__[gometalinter]__
* [aligncheck] ActorUnavailable not declared by package yue
* [gotype] invalid operation: aerr (variable of type *invalid type) has no field or method Is
* [gotype] undeclared name: yue